### PR TITLE
feat(srtLoader): Implement SRT file loader

### LINKED ISSUE #293: Add support for SRT (SubRip Subtitle) file format
### What
Implement a loader for SRT files in Replexica.

### Why
To support video editors and translators working with video subtitles.

### How
1. Create a new loader in `packages/cli/src/loaders/srt.ts`
2. Implement parsing logic for SRT structure
3. Add two unit tests in `packages/cli/src/loaders/index.spec.ts`:
   - Test parsing a simple SRT file
   - Test handling multi-line subtitles in SRT entries
4. Update `@replexica/spec` to include SRT format
5. Generate a changeset using `pnpm new` for both 'replexica' and '@replexica/cli' packages
6. In the pull request, include a link to the SRT specification used.

### After completing the task
Noticed a file format Replexica doesn't yet support? Please create a new GitHub ticket (if it doesn't yet exist!), using the format/structure of this GitHub issue.

### PR DESCRIPTION
**Pull request by partik03**
Should close #293 

- feat: Implement a new createSrtLoader() function in packages/cli/src/loaders/srt.ts to handle loading and saving SRT subtitle files
- test: Add two unit tests in packages/cli/src/loaders/index.spec.ts to validate parsing of simple SRT files 
- docs: Update @replexica/spec to include the SRT file format specification
- chore: Generate changelogs for both 'replexica' and '@replexica/cli' packages using pnpm new

### CODE REVIEW

**Comment by partik03** [comment]
@maxprilutskiy kindly review this one

**Comment by maxprilutskiy** [comment]
@partik03 looks good, please fix the build error (make sure the lockfile is in sync with package.json) and let's merge this one!

Great job!

**Comment by partik03** [comment]
@maxprilutskiy done

File: packages/cli/src/loaders/index.spec.ts [hunk 1]
```diff
@@ -859,6 +859,107 @@ user.password=Contraseña
...
+          text: 'Hello!',
+        },
+        '1': {
+          startTime: '00:00:01,000',
+          startSeconds: 1,
+          endTime: '00:00:02,000',
+          id: "2",
+          endSeconds: 2,
+          text: 'World!',
+        },
```
Comment (maxprilutskiy) [inline packages/cli/src/loaders/index.spec.ts]: can you make it work like this:

```suggestion
 '1#00:00:00,000-00:00:01,000': 'Hello!',
 '2#00:00:01,000-00:00:02,000': 'World!',
```
or similar, to make sure we pass forward only the data that needs to be localized?

e.g. we do that for flutter's metadata or for HTML's tags/attributes.

Comment (partik03) [reply packages/cli/src/loaders/index.spec.ts]: on it

Comment (partik03) [reply packages/cli/src/loaders/index.spec.ts]: done

**Comment by partik03** [comment]
@maxprilutskiy done now check once

**Pull request merged**